### ISSUE TITLE
if debug alone is called, debug never appears

### ISSUE THREAD

**Issue by retrogradeorbit** [epiccastle/spire#72]
debug does not appear until next module update output triggers it...

output module needs some rework...

**Comment by retrogradeorbit** [comment]
```
$ ./spire -e '(debug nil)'
nil
```
Note no debug line

```
$ ./spire -e '(debug nil)(shell {:cmd "true"})'
:1 (shell {:cmd "true"})                                                                                          
--------- :1 local (debug nil) ---------
nil
----------------------------------------
:1 (shell {:cmd "true"}) local                                                                                    
{:err "", :exit 0, :out "", :out-lines [""], :result :ok}
```

debug now appears

**Comment by retrogradeorbit** [comment]
fixed in 0.1.0-alpha.14

```
$ ./spire -e '(debug nil)'
--------- :1 local (debug nil) ---------
nil
----------------------------------------
nil
```
